tools/adjust-package-exports.mts

```diff
@@ -1,12 +1,12 @@
 import { readFile, writeFile } from 'node:fs/promises';
-import { PackageJson } from 'type-fest';
```

Comment:
I know there's a reason, but you say "upgrade"...how does this upgrade?

Reply:
They removed the string literal keys I was using in the latest version. Replaced on our side with `string`, I guess it's a bit more dynamic than expected.